crates/biome_configuration/src/formatter.rs

```diff
@@ -111,58 +107,6 @@ impl FromStr for FormatterConfiguration {
     }
 }
 
-pub fn to_format_settings(
```

Comment:
These conversion logics (`to_***_settings`) are moved in `crates/biome_service/src/settings.rs`